react/features/base/lib-jitsi-meet/native/polyfills-browser.js

```diff
@@ -371,7 +371,7 @@ function _visitNode(node, callback) {
     // - Strophe
     global.clearTimeout = BackgroundTimer.clearTimeout.bind(BackgroundTimer);
     global.clearInterval = BackgroundTimer.clearInterval.bind(BackgroundTimer);
-    global.setInterval = BackgroundTimer.setInterval.bind(BackgroundTimer);
-    global.setTimeout = BackgroundTimer.setTimeout.bind(BackgroundTimer);
+    global.setInterval = (fn, ms = 0) => BackgroundTimer.setInterval(fn, ms);
```

Comment:
What does it mean to repeatedly execute a function with zero milliseconds between executions? Indeed, https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/setInterval documents the `delay`/`ms` argument as required. Did you really get an error about `setInterval`?

Reply:
No, sorry, I was too quick unifying them, but didn't think it through enough. Fixed, thanks!